package.json

```diff
@@ -185,6 +185,6 @@
     "ff esr"
   ],
   "resolutions": {
-    "@mapbox/mapbox-gl-style-spec": "13.14.0"
+    "@mapbox/mapbox-gl-style-spec": "13.23.1"
```

Comment:
Do we still have to keep this around?

Reply:
what do You mean? If we can drop the whole resolution? Maybe. won't test that within that thing here. was ugly enough

Reply:
yes i meant to drop the resolution but i agree that's not in scope here.